txpool/account.go

```diff
@@ -19,20 +19,15 @@ type accountsMap struct {
 
 // Intializes an account for the given address.
 func (m *accountsMap) initOnce(addr types.Address, nonce uint64) *account {
-	a, _ := m.LoadOrStore(addr, &account{})
+	a, _ := m.LoadOrStore(addr, &account{
+		enqueued:    newAccountQueue(),
```

Comment:
Do we have a test to check that the fix helped?

Reply:
Added it 9d62607b5734071734f241fff023aab89c7a9068

Please also check the *Additional Comments* section in the description.